generators/tests/org/jetbrains/kotlin/generators/arguments/GenerateGradleOptions.kt

```diff
@@ -73,7 +73,7 @@ fun generateKotlinGradleOptions(withPrinterToFile: (targetFile: File, Printer.()
     // generate jvm interface
     val jvmInterfaceFqName = FqName("org.jetbrains.kotlin.gradle.dsl.KotlinJvmOptions")
     val jvmOptions = gradleOptions<K2JVMCompilerArguments>()
-    withPrinterToFile(file(srcDir, jvmInterfaceFqName)) {
+    withPrinterToFile(file(apiSrcDir, jvmInterfaceFqName)) {
```

Comment:
Maybe we should be consistent and generate JS/JS-Dce options as well into api artefact :thinking:

Reply:
Agree, done in 911a97e.